CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [3.2.2] - July 10, 2024
```

Comment:
With the Minor bump, I'd suggest 3.2.0. What do you think?

```suggestion
## [3.2.0] - July 10, 2024
```

Reply:
If you agree, then no need to change your branch name, of course.

Reply:
My bad. While upgrading the minor version, I forgot to reset the patch. Sorry! Let me fix this!